src/rules/extensions.js

```diff
@@ -34,7 +36,9 @@ function buildProperties(context) {
       }
 
       // If this is not the new structure, transfer all props to result.pattern
-      if (obj.pattern === undefined && obj.ignorePackages === undefined) {
+      if (obj.pattern === undefined &&
```

Comment:
is this the style the linter wants? i'd expect:
```js
if (
  obj.pattern === undefined &&
  obj.ignorePackages === undefined &&
  obj.commonjs === undefined
) {
```

Reply:
It didn't complain about that style, just with the length of the line when I had them all inline. Changed.